README.md

```diff
@@ -37,7 +43,7 @@ options:
 ### Example
 
 ```
-$ elicznik freddy@example.com secretpassword 2022-07-20
+$ elicznik freddy@example.com secretpassword optional_site_identifier 2022-07-20
```

Comment:
Let's keep the example in the readme simple and skip the site parameter here.  Most users only deal with a single site.